driver_cpl/shr/seq_flds_mod.F90

```diff
@@ -1335,6 +1335,15 @@ subroutine seq_flds_set(nmlfile, ID)
      attname  = 'Si_ifrac'
      call metadata_set(attname, longname, stdname, units)
 
+     ! Sea ice basal pressure
+     call seq_flds_add(i2x_states,"Si_bpress")
```

Comment:
This may need to be in an ifdef if it is not compatible with our cice component @mvertens what do you think?